### PR TITLE
Bump to PHP 7

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref #22 

### Changes

- use local PHPUnit
    - easier to use with various PHP versions, autoloaded (`TestCase` classes)
- use `[]` instead of `array()`
- use `implode()` instead of `join()`
- use strict types
- use return types
- drop random empty spaces
- add `composer.lock` to `.gitignore`, since not commited

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
   - 7.0
   - 7.1
 
 before_script:
   - travis_retry composer self-update
-  - travis_retry composer install --no-interaction --prefer-source --dev
```
Comment (TomasVotruba) [inline .travis.yml:13]: `--dev` is on by default for years

File: phpunit.xml [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
 <phpunit bootstrap="vendor/autoload.php"
-         backupGlobals="false"
-         backupStaticAttributes="false"
          colors="true"
          verbose="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false">
```
Comment (TomasVotruba) [inline phpunit.xml:11]: All these values are default ones, apart `backupGlobals`. But I think we don't need that either here, this code is simple.

**Comment by gherkins** [comment]
👍

**Comment by TomasVotruba** [comment]
:+1:  Wow, thanks a lot!

I'm happy when projects and their maintainers are moving to PHP 7.

**Pull request merged**